concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -397,8 +490,20 @@ exportSections dbDir outDir chunkSize genIndex startHeight blockIndex lastWritte
         then
             openReadOnlyDatabase treeStateDir >>= \case
                 Nothing -> do
-                    putStrLn "Tree state database could not be opened"
-                    return (True, Empty)
+                    KonsensusV1.openReadOnlyDatabase treeStateDir >>= \case
```

Comment:
Could you comment here that you do "try and catch" approach and note the reasons a bit, so it's easier to maintain in the future?

Reply:
Documented this.